README.md

```diff
@@ -57,7 +56,7 @@ dotnet test
 
 This includes a set of acceptance tests that run
 against an actual mountebank instance. In order for those tests to succeed, mountebank 
-will need to be run with the `--mock` option provided. See http://www.mbtest.org/docs/api/overview#get-imposter.
+will need to be run with the `--mock --allowInjection` option provided. See http://www.mbtest.org/docs/api/overview#get-imposter.
```

Comment:
Could you update `docker/mountebank/Dockerfile` to include the `--allowInjection` parameter in its arguments as well (should just need to change the `CMD [ ]` line)? That way the tests will still pass for anyone running mountebank in a container.